src/campbellcontrol/commands/commands.py

```diff
@@ -54,6 +54,10 @@ def __init__(self, group_id: str, serial: str) -> None:
         """
         self.publish_topic = f"{group_id}/cc/{serial}/{self.command_name}"
         self.response_topic = f"{group_id}/cr/{serial}/{self.command_name}"
+
+        if options and "response_suffix" in options:
```

Comment:
The handling of this custom option does feel like a kludge but didn't want to make a change more generic until we see which commands don't follow the predictable publish-response pattern we otherwise assume - I've only seen two but haven't really close read the spec yet